src/app/shared/issue-tables/IssuesDataTable.ts

```diff
@@ -87,12 +78,10 @@ export class IssuesDataTable extends DataSource<Issue> implements FilterableSour
               return issue.issueOrPr !== 'PullRequest' && issue.assignees.length === 0;
             });
           }
+
           // Dropdown Filters
-          data = data.filter(applyDropdownFilter(this.dropdownFilter));
+          data = applyDropdownFilter(this.dropdownFilter, data);
 
-          if (this.sort !== undefined) {
-            data = getSortedData(this.sort, data);
-          }
           data = applySearchFilter(this.filter, this.displayedColumn, this.issueService, data);
           this.count = data.length;
 
```

Comment:
Over here, could we do `applyDropdownFilter`, then `applySearchFilter` and finally `applySort`?

Reply:
Order of applying different filters and sorting has been updated.